src/defaultsettings.cpp

```diff
@@ -488,6 +489,7 @@ void set_default_settings()
 #endif
 	// Altered settings for Android
 #ifdef __ANDROID__
+	settings->setDefault("enable_post_processing", "false");
```

Comment:
You could put this together with the Android default value for debanding, since debanding is a postprocessing effect.

https://github.com/minetest/minetest/blob/6cfdcfbc3744cf04f2335e2776e3fd898cd7b195/src/defaultsettings.cpp#L507

Reply:
Done.